src/pykiso/cli.py

```diff
@@ -112,10 +125,11 @@ def check_file_extension(
 )
 @click.option(
     "-l",
-    "--log-path",
+    "--log-paths",
```

Comment:
I wouldn't introduce a breaking change on the name just because we are now supporting multiple ones, only the option's help should be extended IMO

Reply:
yes pls no breaking change

Reply:
done